pkg/tools/cli/loadtest/service/insert/insert.go

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package insert
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"reflect"
+	"sync/atomic"
+	"syscall"
+	"time"
+
+	"github.com/vdaas/vald/apis/grpc/gateway/vald"
+	"github.com/vdaas/vald/apis/grpc/payload"
+	"github.com/vdaas/vald/internal/errgroup"
+	"github.com/vdaas/vald/internal/errors"
+	"github.com/vdaas/vald/internal/log"
+	"github.com/vdaas/vald/internal/net/grpc"
+	"github.com/vdaas/vald/internal/safety"
+	"github.com/vdaas/vald/pkg/tools/cli/loadtest/assets"
+)
+
+type insert struct {
+	eg          errgroup.Group
+	client      grpc.Client
+	addr        string
+	concurrency int
+	dataset     string
+	req         []*payload.Object_Vector
+}
+
+func New(opts ...Option) (i *insert, err error) {
+	i = new(insert)
+	for _, opt := range append(defaultOpts, opts...) {
+		if err = opt(i); err != nil {
+			return nil, errors.ErrOptionFailed(err, reflect.ValueOf(opt))
+		}
+	}
+
+	i.eg = errgroup.Get()
+	return i, nil
+}
+
+func (i *insert) Prepare(ctx context.Context) error {
+	fn := assets.Data(i.dataset)
+	if fn == nil {
+		return fmt.Errorf("dataset load funciton is nil: %s", i.dataset)
```

Comment:
```suggestion
		return errors.Errorf("dataset load funciton is nil: %s", i.dataset)
```

Could you please replace fmt pacakge to internal/errors replace